install.py

```diff
@@ -221,9 +224,8 @@ def before_checks(inst, remove, usage):
             sock.close()
         if errcode == 0:  # no error, the DbServer is up
             sys.exit('There is DbServer running on port %d from a previous '
-                     'installation. Please run `%s`. '
-                     'If it does not work, try `sudo fuser -k %d/tcp`' %
-                     (inst.DBPORT, cmd, inst.DBPORT))
+                     'installation. Please change the port or run `%s`'
+                     (inst.DBPORT, cmd))
 
```

Comment:
Can it be helpful to have two different messages for user and server installation?
For a server or devel server installation, shouldn't the dbserver be stopped instead of changing the port?

Reply:
This is actually a good idea. Will do.